docs/bucket/notifications/README.md

```diff
@@ -431,6 +431,7 @@ An example of Redis configuration is as follows:
 "redis": {
     "1": {
         "enable": true,
+	    "format": "namespace",
```

Comment:
Now it is misaligned https://github.com/minio/minio/blob/18726fe8081d1d85d714dbe77e6b395b0ce5508c/docs/bucket/notifications/README.md#Redis